eventdata/parameter_sources/elasticlogs_bulk_source.py

```diff
@@ -81,7 +81,13 @@ def __init__(self, track, params, **kwargs):
         self.orig_args = [track, params, kwargs]
         self._indices = track.indices
         self._params = params
-        self._randomevent = RandomEvent(params)
+        # we could also do `kwargs.get("random_event, RandomEvent(params))` but that would call the constructor eagerly
```

Comment:
I guess there's a `"` missing here, i.e. `kwargs.get("random_event", RandomEvent(params))`?

Reply:
Thanks! Fixed in https://github.com/elastic/rally-eventdata-track/pull/45/commits/0bd74640ee70041dd10b250a8b5ffcf218a40e23.